mix.exs

```diff
@@ -27,6 +27,7 @@ defmodule ABI.Mixfile do
   defp deps do
     [
       {:ex_doc, "~> 0.14", only: :dev, runtime: false},
+      {:poison, "~> 3.1", only: [:dev, :test]},
```

Comment:
Should this be just `test`?

Reply:
`:test` is for running `mix test` in the project scope; `:dev` is for running e.g. `iex -S mix` in the project scope. It's handy to have any libraries depended on by your doctests in both environments—`:test` to run them, and `:dev` to create and debug them.

N.B. when a Mix project is built as a dependency of another project, it's always being built with `Mix.env == :prod` regardless of the parent project's Mix env. (Yes, even though the built artifacts of the dep end up in `_build/dev/lib` instead of `_build/prod/ebin`—that's namespacing for the sake of the parent project.) `:poison` will not be transitively depended-on when something depends on `:abi`.